packages/phoenix-evals/src/phoenix/evals/executors.py

```diff
@@ -225,7 +225,10 @@ def termination_handler(signum: int, frame: Any) -> None:
         original_handler = signal.signal(self.termination_signal, termination_handler)
         outputs = [self.fallback_return_value] * len(inputs)
         execution_details = [ExecutionDetails() for _ in range(len(inputs))]
-        progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format)
+        if self.tqdm_bar_format is None:
+            progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format, disable=True)
+        else:
+            progress_bar = tqdm(total=len(inputs), bar_format=self.tqdm_bar_format)
```

Comment:
nit (same for the other location)
```suggestion
        progress_bar = tqdm(
            total=len(inputs), 
            bar_format=self.tqdm_bar_format, 
            disable=self.tqdm_bar_format is None,
        )
```